### ISSUE TITLE
getDefiningElement NullPointerException

### ISSUE THREAD

**Issue by nicobustillos** [wt-io-it/odoo-pycharm-plugin#30]
Plugin Version: 0.5.12
IDEA Version: PyCharm 2021.3.3

<details><summary>java.lang.NullPointerException</summary>

```
java.lang.NullPointerException
	at java.base/java.util.Optional.orElseThrow(Optional.java:408)
	at at.wtioit.intellij.plugins.odoo.models.impl.OdooModelImpl.getDefiningElement(OdooModelImpl.java:94)
	at at.wtioit.intellij.plugins.odoo.models.impl.OdooModelImpl.lambda$getDefiningElement$4(OdooModelImpl.java:65)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:852)
	at at.wtioit.intellij.plugins.odoo.models.impl.OdooModelImpl.getDefiningElement(OdooModelImpl.java:49)
	at at.wtioit.intellij.plugins.odoo.OdooGoToDeclarationHandler.getOdooModel(OdooGoToDeclarationHandler.java:63)
	at at.wtioit.intellij.plugins.odoo.OdooGoToDeclarationHandler.getOdooModel(OdooGoToDeclarationHandler.java:55)
	at at.wtioit.intellij.plugins.odoo.OdooGoToDeclarationHandler.getGotoDeclarationTarget(OdooGoToDeclarationHandler.java:32)
	at com.intellij.codeInsight.navigation.actions.GotoDeclarationHandlerBase.getGotoDeclarationTargets(GotoDeclarationHandlerBase.java:14)
	...
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.lambda$transferToBgThread$1(NonBlockingReadActionImpl.java:407)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	...
	at java.base/java.lang.Thread.run(Thread.java:829)
```
</details>

**Comment by bigbear3001** [comment]
this has been fixed in 0.6.9 (pending JetBrains approval). if you are eager to test the new version, you can use the jar file from https://github.com/wt-io-it/odoo-pycharm-plugin/tree/v0.6.9

**Comment by ap-wtioit** [comment]
0.6.9 has been approved and is available in the marketplace
